src/components/Panel/PanelContext.tsx

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 
 export interface PanelContextProps {
   panel?: string;
+  getPanelNode(): HTMLElement | null;
```

Comment:
Можно добавить игнор, и расширять прописи типы компонента от интерфеса

```suggestion
  /**
   * @ignore
   */
  panel?: string;
  /**
   * @ignore
   */
  getPanelNode(): HTMLElement | null;
```